README.md

```diff
@@ -1,2 +1,51 @@
-Hello-World
-In this project, you will set up a "Hello world" repository.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This project is my first ever on microverse and its a linter setup as well as a hello world
+
+![screenshot](./app_screenshot.png)
+
+This is a simple hello world project
+
+## Built With
+
+- HTML
+- CSS
+
+<!-- ## Live Demo
+
+[Live Demo Link](https://livedemo.com) -->
+
+
+## Getting Started
+
+
+you can clone and play around 
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/BereketRetta)
+- Twitter: [@twitterhandle](https://twitter.com/bekiopia)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/bereketretta)
```

Comment:
- [ ] In the `Authors` section kindly add your social links instead of the ones in the example

Reply:
This is my own links